src/index.ts

```diff
@@ -54,7 +54,9 @@ const validateDevcardIdAsUUID = (devcard_id: string): boolean => {
 		const branch = core.getInput('commit_branch')
 		const message = core.getInput('commit_message')
 		const filename = core.getInput('commit_filename')
-		const dryrun = core.getBooleanInput('dryrun')
+		const email = core.getInput('committer_email')
+		const name = core.getInput('committer_name')
+		const dryrun = false
```

Comment:
```suggestion
		const dryrun = false
```

Don't set dryrun to static false

Reply:
Ahh sorry, I've changed that when testing locally